gem/lib/earlgrey/configure_earlgrey.rb

```diff
@@ -13,19 +13,44 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-# global export for unqualified use in Pods file
-def configure_for_earlgrey(*args)
-  EarlGrey.configure_for_earlgrey(*args)
+require 'colored'
+require 'fileutils'
+require 'xcodeproj'
+
+def configure_for_earlgrey
+  raise 'configure_for_earlgrey is deprecated, please remove.'
 end
 
 module EarlGrey
+  XCScheme = Xcodeproj::XCScheme
+  EnvironmentVariable = XCScheme::EnvironmentVariable
+
+  XCSCHEME_EXT = '*.xcscheme'.freeze
```

Comment:
good to know, thanks! looks like rubocop will warn `.freeze` on prior ruby2.3, I wanted to at least make sure this works with the built-in ruby as most users won't upgrade their Ruby just for installing a gem.